src/c++/vdo/base/repair.c

```diff
@@ -36,7 +36,7 @@
 #define VDO_USE_NEXT
 #endif
 #else /* RHEL_RELEASE_CODE */
-#if (LINUX_VERSION_CODE > KERNEL_VERSION(6, 10, 0))
+#if (LINUX_VERSION_CODE > KERNEL_VERSION(6, 11, 0)) || (LINUX_VERSION_CODE == KERNEL_VERSION(6, 11, 0))
 #define VDO_USE_NEXT
```

Comment:
Can you really not spell this as (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 11, 0))?
You could also spell it as !(LINUX_VERSION_CODE < KERNEL_VERSION(6, 11, 0)).
It shouldn't need two clauses, though.